pkg/utils/utils.go

```diff
@@ -373,3 +377,37 @@ func Chroot(path string) (func() error, error) {
 		return syscall.Chroot(".")
 	}, nil
 }
+
+func setVfsAdminMac(iface *sriovnetworkv1.InterfaceExt) error {
+	glog.Infof("setVfsAdminMac(): device %s", iface.PciAddress)
+	pfLink, err := netlink.LinkByName(iface.Name)
+	if err != nil {
+		glog.Errorf("setVfsAdminMac(): unable to get PF link for device %+v %q", iface, err)
+		return err
+	}
+	vfs, err := dputils.GetVFList(iface.PciAddress)
+	if err != nil {
+		return err
+	}
+	for _, addr := range vfs {
+		vfID, err := dputils.GetVFID(addr)
+		if err != nil {
+			glog.Errorf("setVfsAdminMac(): unable to get VF id %+v %q", iface.PciAddress, err)
+			return err
+		}
+		vfName := tryGetInterfaceName(addr)
+		vfLink, err := netlink.LinkByName(vfName)
+		if err != nil {
+			glog.Errorf("setVfsAdminMac(): unable to get VF link for device %+v %q", iface, err)
+			return err
+		}
+		if err := netlink.LinkSetVfHardwareAddr(pfLink, vfID, vfLink.Attrs().HardwareAddr); err != nil {
+			return err
+		}
+		if err = BindDefaultDriver(addr); err != nil {
```

Comment:
@Mmduh-483 I just noticed this line, why do we need to bind default driver here? I assume when the VFs are first provisioned, they all bind to its corresponding vf drivers.

Reply:
So set admin mac and rebind the VF driver set the node_guid. This is the issue we wanted to fix in the first place. 
We need node_guid to so that it will work with rdma_cm

Reply:
This line only bind default driver, not unbind, does it actually work with rdma_cm ?

Reply:
good point @Mmduh-483  is retesting it again. It maybe a bug

Reply:
once the unbind/bind is verified, we can see how to move the unbind/bind logic in mlx plugin, and leave the admin MAC settings in generic plugin.